lua/femaco/edit.lua

```diff
@@ -170,6 +170,16 @@ M.edit_code_block = function()
     buffer = 0,
     callback = function()
       local lines = vim.api.nvim_buf_get_lines(float_bufnr, 0, -1, true)
+
+      local lines_changed = false
```

Comment:
[nit] nice, looks good! maybe could just break this out to a function checking equivalence of two tables which could make this a little more readable